rubrics/impl/src/main/java/org/sakaiproject/rubrics/logic/RubricsServiceImpl.java

```diff
@@ -1255,5 +1256,13 @@ public void deleteSiteRubrics(String siteId) {
             log.error("Rubrics: error trying to delete rubric -> {}" , e.getMessage());
         }
     }
+    
```

Comment:
Is this for checking if the export option is enabled?

Maybe this?

serverConfigurationService.getBoolean(RUBRICS_EXPORT_PDF, false);

Reply:
Done.